sdk/identity/identity/src/credentials/azurePipelinesCredential.ts

```diff
@@ -27,37 +27,38 @@ export class AzurePipelinesCredential implements TokenCredential {
    * @param tenantId - tenantId associated with the service connection
    * @param clientId - clientId associated with the service connection
    * @param serviceConnectionId - id for the service connection, as found in the querystring's resourceId key
+   * @param systemAccessToken - The Azure pipeline's <see href="https://learn.microsoft.com/azure/devops/pipelines/build/variables?view=azure-devops%26tabs=yaml#systemaccesstoken">System.AccessToken</see> value.
    * @param options - The identity client options to use for authentication.
    */
   constructor(
     tenantId: string,
     clientId: string,
     serviceConnectionId: string,
-    options?: AzurePipelinesCredentialOptions,
+    systemAccessToken: string,
+    options?: AzurePipelinesCredentialOptions
   ) {
-    if (!clientId || !tenantId || !serviceConnectionId) {
+    if (!clientId || !tenantId || !serviceConnectionId || !systemAccessToken) {
       throw new CredentialUnavailableError(
-        `${credentialName}: is unavailable. tenantId, clientId, and serviceConnectionId are required parameters.`,
+        `${credentialName}: is unavailable. tenantId, clientId, serviceConnectionId and systemAccessToken are required parameters.`
       );
     }
     this.identityClient = new IdentityClient(options);
     checkTenantId(logger, tenantId);
     logger.info(
-      `Invoking AzurePipelinesCredential with tenant ID: ${tenantId}, clientId: ${clientId} and service connection id: ${serviceConnectionId}`,
+      `Invoking AzurePipelinesCredential with tenant ID: ${tenantId}, clientId: ${clientId} and service connection id: ${serviceConnectionId}`
     );
 
-    if (clientId && tenantId && serviceConnectionId) {
+    if (clientId && tenantId && serviceConnectionId && systemAccessToken) {
```

Comment:
We don't need this `if` statement here because we already ensure these all exist in line 40